### ISSUE TITLE
Wheel Covers

### ISSUE THREAD

**Issue by smithbeuvic** [UVicFH/Home#26]
### Context

Our tire covers need to be remade or altered to fit appropriately. These preserve the tires and look good. 

### Outcomes

 - Alterations to the current wheel covers such that they completely cover the wheels and can be put on/removed with ease.
or
 - New wheel covers made from an aesthetically pleasing but durable material such that they completely cover the wheels and can be put on/removed with ease.

### Constraints

 - Budget is $50
 - Car must be able to roll around with them on
 - Must look Awesome

### Steps to reproduce behavior

 - Measure the dimensions required for the tire sleeve
 - Decide on a good securing/tightening mechanism
 - Decide what material will be used
 - Decide how these will be manufactured
 - Buy everything and make everything

**Comment by chadjmccolm** [comment]
Here's my take on it. Unsure of material. I'm against using a pull cord for tightening because I don't want it to get in the way but don't like the current attachment with velcro because I don't want the tires sitting on a hump for extended periods of time in transport etc. so I suggest a elastic on the inside only. I think filling the outside edge also gives us a good place to put some sort of eye candy. 

![photoscan](https://user-images.githubusercontent.com/22900772/31595113-38e2f16e-b1ee-11e7-977e-e286997eb355.jpg)

**Comment by chadjmccolm** [comment]
Just realized that the inner face needs to change to something similar to the body in shape so that it can slip over the tire. How far the elastic we use stretches will determine how wide it is.

**Comment by chadjmccolm** [comment]
I went and talked to the lady at fabric land. She recommended using marine vinyl or vinyl coated solids. The elastic band idea was discarded because it'll make it hard to have a perfect fit and still be able to get the thing on. It'd be nice to put grommets through it though so we can tie up the back with paracord.

**Comment by smithbeuvic** [comment]
![capture](https://user-images.githubusercontent.com/28762035/31839544-704afb82-b596-11e7-977c-de1e27f8f8c5.PNG)
 Like this one piece design.

**Comment by chadjmccolm** [comment]
Yeah. Either with elastic or cord to tie up the back. Otherwise if we want to show off how cool our wheels are we can do a similar pattern to the one we have now but I'd rather the part along the tire be sewn than velcrod. Then we can do a set of grommets along each side or wearing but I think it'll be easier to build and look cooler with grommets and tied up with Paracord or something

**Comment by chadjmccolm** [comment]
I actually really like the idea of having it open faced front and back like it is now and tying with coloured cord. We could do a blue white scheme between the cord and the fabric

**Comment by smithbeuvic** [comment]
fully agree, I like the wheel showing.
